pkgs/applications/editors/emacs/generic.nix

```diff
@@ -45,6 +44,8 @@
   else if withMotif then "motif"
   else if withAthena then "athena"
   else "lucid")
+, systemdSupport ? stdenv.isLinux
```

Comment:
Other Emacs options are in the style of `withFoo`, `systemdSupport` is inconsistent with other emacs options.
A better naming would be `withSystemd`.

There is no global consensus in nixpkgs, but at least we can try to be locally consistent.